### DISCUSSION TITLE
Need a remix.run template with theme-ui

### DISCUSSION THREAD

**Discussion by NetLancer** [system-ui/theme-ui#2231] [Q&A] [answered]
Hi guys! :) Does anyone know such a template for remix framework? There seems to be none ready made with theme-ui in their repo :(

**Comment by lachlanjc** [comment, marked as answer]
Not aware of any yet, & haven't tried the two together myself yet! If anyone does find or build an example, would love to link to it or bring a simplified version into this repo.

**Reply by NetLancer** [reply]
Yeah, would be great if some cute guy build one, there are not many styling/ui solutions i've found after looking for a (long) while and theme-ui's turned to be one of them. Doing (especially) fullstack i see it a best fit to have clean & semantic markup and a flexibility to forge just elements you need

**Comment by guatedude2** [comment]
Hey guys, cute guy here, thought you'd find this useful

https://github.com/remix-run/remix/pull/3637

**Reply by guatedude2** [reply]
@NetLancer @lachlanjc Remix example is now merged

**Reply by NetLancer** [reply]
Great! Many-many thanks for such an endowment:)

**Reply by lachlanjc** [reply]
Oh, amazing! Thank you for contributing that.

**Comment by Centerworx** [comment]
@guatedude2 This example no longer works & it has been removed from Remix site. Any suggestions to get it to work? 

It will load, then disappear. With a hydration mismatch error.

**Reply by lachlanjc** [reply]
To clarify, you're running into issues with this example? https://github.com/remix-run/examples/tree/main/theme-ui

**Comment by guatedude2** [comment]
I’ll have to check

**Comment by guatedude2** [comment]
@Centerworx, I just tested the example and it seems to be working correctly. Can you provide more details of the errors? If you can include node version, terminal logs and console logs that would be very helpful.
